tests/integration/service_test.go

```diff
@@ -101,4 +102,42 @@ var _ = Describe("odoServiceE2e", func() {
 	// 	})
 	// })
 
+	Context("When working from outside a component dir", func() {
```

Comment:
the context says "when working **outside** a component dir" but then "JustBeforeEach" switches the current dir to the context dir  :-/

Reply:
That is for the sake of copying the nodejs example and creating a component from it. But for the test, we have made sure that we cd to `originalDir` https://github.com/openshift/odo/pull/1920/files#diff-2bf31fdcc572d1ff24f1215e18667e08R137-R140

Reply:
ok, I missed that. 
/lgtm